### ISSUE TITLE
test_stdlib fails on Windows Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8755]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8755|
|Type|enhancement|
|Created|2016-08-14 09:31:00Z|
|Branch|https://github.com/twisted/twisted/tree/8755-rodrigc-stdlib-py3|

twisted.logger.test.test_stdlib.STDLibLogObserverTests.test_basicFormatRenderered fails
on Windows with Python 3.5:

```
[FAIL]
Traceback (most recent call last):
  File "c:\projects\twisted\build\py35-coverage-windows\lib\site-packages\twisted\logger\test\test_stdlib.py", line 174, in test_basicFormatRendered
    repr(output))
  File "c:\projects\twisted\build\py35-coverage-windows\lib\site-packages\twisted\trial\_synctest.py", line 388, in assertTrue
    super(_Assertions, self).assertTrue(condition, msg)
  File "C:\PYTHON35-x64\Lib\unittest\case.py", line 677, in assertTrue
    raise self.failureException(msg)
twisted.trial.unittest.FailTest: False is not true : 'INFO:twisted:Hello, dude!\r\n'
twisted.logger.test.test_stdlib.STDLibLogObserverTests.test_basicFormatRendered
```

<details><summary>Searchable metadata</summary>

```
trac-id__8755 8755
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8755_rodrigc_stdlib_py3 8755-rodrigc-stdlib-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1471167060606254 1471167060606254
changetime__1471515079316695 1471515079316695
version__None None
owner__Craig_Rodrigues__rodrigc_____ Craig Rodrigues <rodrigc@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/494

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

chnages look good. Please merge

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set owner to @rodrigc<br>@rodrigc set status to `closed`|
|-|-|

In changeset f6766ae23afdf36e56a3c978d0f47ff34670f250
```
#!CommitTicketReference repository="" revision="f6766ae23afdf36e56a3c978d0f47ff34670f250"
Merge pull request #494 from twisted/8755-rodrigc-stdlib-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8755

Fix test_stdlib.py on Windows with Python 3.5
```
